### PR TITLE
Add telemetry to dashboard. Use icons on dashboard.

### LINKED ISSUE #147: Show telemetry data in admin overview
Mostly a display change.

### PR DESCRIPTION
**Pull request by pmtischler**
Fixes #147

### CODE REVIEW

File: client/tools/prober.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 import datagen
 import flightsim
-from interop import AsyncClient, InteropError, ServerInfo, Telemetry, StationaryObstacle, MovingObstacle
+from interop import AsyncClient, InteropError, Telemetry, StationaryObstacle, MovingObstacle
```
Comment (prattmic) [inline client/tools/prober.py]: Can you split the  client and prober changes into a different commit? They are unrelated to the frontend.

File: server/auvsi_suas/static/auvsi_suas/components/team-status/team-status.html [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
+            <h6>Lat: {{teamStatusCtrl.telemetry.latitude | number : 6}}</h6>
         </div>
-        <div class="small-6 columns">
-            <h6>Active: {{teamStatusCtrl.team.active}}</h6>
-            <h6>In Air: {{teamStatusCtrl.team.in_air}}</h6>
+        <div class="small-4 columns">
+            <h6>Lon: {{teamStatusCtrl.telemetry.longitude | number: 6}}</h6>
+        </div>
+        <div class="small-4 columns">
+            <h6>Alt: {{teamStatusCtrl.telemetry.altitude_msl | number: 6}}</h6>
```
Comment (prattmic) [inline server/auvsi_suas/static/auvsi_suas/components/team-status/team-status.html]: "ft MSL" would be helpful here.

File: server/auvsi_suas/static/auvsi_suas/components/team-status/team-status.html [hunk 3]
```diff
@@ -1,13 +1,34 @@
...
+        </div>
+    </div>
+    <div class="row">
+        <div class="small-12 columns">
+            <span class="team-status-icon label" ng-if="teamStatusCtrl.team.on_clock"
+                  title="On Mission Clock">
+                &#128336;
```
Comment (prattmic) [inline server/auvsi_suas/static/auvsi_suas/components/team-status/team-status.html]: Could you add a comment along the lines of "CLOCK FACE ONE OCLOCK' (U+1F550)". It took me quite a while to figure out which character this is.

File: server/auvsi_suas/static/auvsi_suas/components/team-status/team-status.html (hunk 2 above)
Comment (pmtischler) [inline server/auvsi_suas/static/auvsi_suas/components/team-status/team-status.html]: Done.

File: server/auvsi_suas/static/auvsi_suas/components/team-status/team-status.html (hunk 3 above)
Comment (pmtischler) [inline server/auvsi_suas/static/auvsi_suas/components/team-status/team-status.html]: Done.

**Pull request merged**